### DISCUSSION TITLE
Fullscreen issue and dpi awarness

### DISCUSSION THREAD

**Discussion by ARTony8** [TomSchimansky/CustomTkinter#1443] [Q&A]
I have a simple application (I'm using WINDOWS 11) with a simple window that displays 'Hello world', and I set it to fullscreen when i click on F11 using :  `self.attributes('-fullscreen', True)`

The problem is that when I click on F11, the window get fullscreen but not completely. The window just have a different size (not fullscreen) and is stucked on the left-top of the screen. Like : 
![issue](https://user-images.githubusercontent.com/97738245/230945430-4b0a7af6-afa0-4c77-9588-ea59663c4698.png)

I did some research and it is due to the automatic dpi awarness of customtkinter. I simply need to put : `ctk.deactivate_automatic_dpi_awareness()` and it's good BUT I got blurry widgets now, due to the the fact that my WINDOWS 11 is setup to be scaled to 130%.

My question is that, is there a way to have a good fullscreen window, without having to use `ctk.deactivate_automatic_dpi_awareness()` ??

**Comment by zSynctic** [comment]
Can you provide your code
